source/NVDAObjects/__init__.py

```diff
@@ -798,6 +798,12 @@ def _get_statusBar(self):
 		"""
 		return None
 
+	def getValueForAriaCurrent(self):
+		"""Gets the value for aria-current. Normally returns False. If this object is current
+		it will return one of the following values: True, "page", "step", "location", "date", "time"
+		"""

```

Comment:
Yeah ok. I'm not 100% comfortable with the name, but I can't think of anything better.